issues/handler_test.go

```diff
@@ -47,7 +47,7 @@ func TestListHandler_ServeHTTP(t *testing.T) {
 			expectedStatus: http.StatusOK,
 			listClient: &fakeClient{
 				issues: []*github.Issue{
-					&github.Issue{
```

Comment:
I know it seems unrelated, but this cleans up a lint warning I was getting from go1.14.